.github/workflows/cd.yml

```diff
@@ -5,19 +5,30 @@ on:
     branches:
       - main
   workflow_dispatch:
-#  pull_request:
+  pull_request:
 
 jobs:
   push-executors:
     runs-on: ubuntu-latest
-    container:
-      image: jinaai/jina:master-standard
-      options: "--entrypoint /bin/bash"
     steps:
+      - name: install git
+        run: |
+          # checkout needs git >= 2.18 to check out repo as git repo and not archive
+          # repo is needed for getting tag
+          sudo apt-get update && sudo apt-get install -y git && sudo apt-get upgrade git
```

Comment:
FYI `ubuntu-latest` should come with Git installed. Unless they changed that

Reply:
Seems it does. The problem was that we were using a custom container. Fixed now. Thanks :)